spec/features/as_a_whole/cangaroo_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe 'Cangaroo', type: :request do
+  class FakeJob < Cangaroo::Job
```

Comment:
@aleph1ow Great idea to add some integration tests!

We should telling the polling infrastructure as well! What do you think about adding a `FakePollJob < Cangaroo::PollJob` spec?